.github/workflows/actions/test-angular-e2e/action.yml

```diff
@@ -8,7 +8,7 @@ runs:
   steps:
     - uses: actions/setup-node@v4
       with:
-        node-version: 18
+        node-version: 22.x
```

Comment:
Angular 20 requires a minimal node version of 20. I verified that version 22 works with all the Angular test apps that we support. By updating it to `22.x`, renovate will be able to update it when necessary instead of keeping it pinned.